### DISCUSSION TITLE
How would one go about finding nicknames in custom commands?

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#4645] [Q&A]
As the title says, is there any known CMI-compatible plugin with placeholders to find the IGN based on a nickname?
I plan to utilize CMI to it's fullest capabilities and create a smooth, user-friendly feeling throughout the server.
(I'm aware duplicate nicks can exist, I'm also aware that duplicate IGNs can exist on the same server.)

To clear up any confusion, I basically mean this;
My IGN: nemmynaomi
My nickname in-game: naomi

My "custom command" interacts with other plugins, those of which may not support CMI's nicknames.
Doing `/example naomi` wouldn't work- as the plugin doesn't recognize anyone by that name, but I'm wondering if I can make it work?

**Comment by CloudeLecaw** [comment]
When doing /cmi info nickname you see the Info about a Player this contains also his real playerName + his uuid and also his namechange history

**Comment by Zrips** [comment]
@nemmynaomi 
Question would be: How do you plan to access players name? 
Custom alias can take and insert players real name or nickname or display name, where each of them could be different one. So you can choose one option from: [playerName] [playerDisplayName] [playerNick]
Tho this does require reference point so you cant simply use some placeholder which would include players name to get players information.

**Comment by mrfloris** [comment]
```
> cmi checkcommand real
info: cmi realname (playerName/nickName)
```
